### PR TITLE
Create scaling-puppet-enterprise.md

### PR DESCRIPTION
**Pull request by npwalker**
Addresses https://github.com/puppetlabs/cs-standards/issues/81

### CODE REVIEW

File: scaling-puppet-enterprise.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Scaling Puppet Enterprise
+
+## Summary
+
+This standard defines how to determine bottlenecks in PE and think about adding compile masters or adding resources to an existing PE installation.
+
+## Expectations
+
+This standard relies on the [Monitoring PE standard]() and assumes the user will have metrics that can be trended over time.
```
Comment (beergeek) [inline scaling-puppet-enterprise.md]: @npwalker can you please put in the link to the monitoring/metrics standard?

File: scaling-puppet-enterprise.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+10K / 3 = 3,333 nodes per compile master 
+
+The magic number for how many nodes a compile master can support changes over time as puppet code is added, catalogs change, and/or facts are added.  When you are running compile masters with more than 1 avg free JRuby then you have capacity to add more nodes to your PE installation.  Another useful calculation is nodes per JRuby.
```
Comment (beergeek) [inline scaling-puppet-enterprise.md]: @npwalker The second sentence sounds a little weird. Can you put it another way as I am a little confused.

Comment (beergeek) [reply scaling-puppet-enterprise.md]: @npwalker can we add something like 'Puppet recommends an average of one free jruby per compile masters. If adding new nodes you should be concerned if this average is less than one'?

**Comment by npwalker** [comment]
Any further comments before a merge?

File: scaling-puppet-enterprise.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Standard Details
+
+### Preferred Option
+
+There are three main bottlenecks in PE:
+
+Catalog compilation
+PostgreSQL ( the backend database to PuppetDB )
+PuppetDB service
```
Comment (abottchen) [inline scaling-puppet-enterprise.md]: You'll need some markdown here.  These are all rendering on the same line.

**Pull request merged**